File: dailyQuestion/2020-07/07-29/golang/solution_n.go
package main

func missingNumber(nums []int) int {
	for i,v := range(nums) {
		if i != v {
			return i
		}
	}
	return len(nums)
}

func main() {
	nums := []int{0, 1, 2, 3, 4, 5, 6, 7, 9}
	result := missingNumber(nums)
	print(result)
}

File: dailyQuestion/2020-07/07-29/python/solution_n.py

# 一次遍历
def missingNumber(nums: [int]) -> int:
    for i,v in enumerate(nums) :
        if i != v :
            return i
    return nums[-1] + 1

if __name__ == "__main__" :
    nums = [0,1,2,3,4,5,6,7,9]
    result = missingNumber(nums)
    print(result)